docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md

```diff
@@ -10,35 +10,48 @@ localization_priority: Priority
-- [Office.context.mailbox.item.getAttachmentsAsync](office.context.mailbox.item.md#getattachmentsasyncoptions-callback--arrayattachmentdetailsjavascriptapioutlookofficeattachmentdetails) - Added a new method that gets an item's attachments in compose mode.
-- [Office.context.mailbox.item.getInitializationContextAsync](office.context.mailbox.item.md#getinitializationcontextasyncoptions-callback) - Added a new function that returns initialization data passed when the add-in is [activated by an actionable message](https://docs.microsoft.com/outlook/actionable-messages/invoke-add-in-from-actionable-message).
-- [Office.context.mailbox.item.getSharedPropertiesAsync](office.context.mailbox.item.md#getsharedpropertiesasyncoptions-callback) - Added a new method that gets an object which represents the sharedProperties of an appointment or message item.
-- [Office.context.mailbox.item.internetHeaders](office.context.mailbox.item.md#internetheaders-internetheadersjavascriptapioutlookofficeinternetheaders) - Added a new property that represents the internet headers on a message item.
-- [Office.context.auth.getAccessTokenAsync](https://docs.microsoft.com/office/dev/add-ins/develop/sso-in-office-add-ins#sso-api-reference) - Added access to `getAccessTokenAsync`, which allows add-ins to [get an access token](https://docs.microsoft.com/outlook/add-ins/authenticate-a-user-with-an-sso-token) for the Microsoft Graph API.
-- [Office.MailboxEnums.AttachmentContentFormat](/javascript/api/outlook/office.mailboxenums.attachmentcontentformat) - Added a new enum that specifies the formatting that applies to an attachment's content.
-- [Office.MailboxEnums.AttachmentStatus](/javascript/api/outlook/office.mailboxenums.attachmentstatus) - Added a new enum that specifies whether an attachment was added to or removed from an item.
-- [Office.MailboxEnums.DelegatePermissions](/javascript/api/outlook/office.mailboxenums.delegatepermissions) - Added a new bit flag enum that specifies the delegate permissions.
-- [Office.MailboxEnums.LocationType](/javascript/api/outlook/office.mailboxenums.locationtype) - Added a new enum that specifies an appointment location's type.
-- [Office.EventType](/javascript/api/office/office.eventtype) - Modified to support AttachmentsChanged, EnhancedLocationsChanged, and OfficeThemeChanged events through addition of `AttachmentsChanged`, `EnhancedLocationsChanged`, and `OfficeThemeChanged` entries respectively.
-- [SupportsSharedFolders manifest element](../../manifest/supportssharedfolders.md) - Added a child element to the [DesktopFormFactor](../../manifest/desktopformfactor.md) manifest element. It defines whether the add-in is available in delegate scenarios.
+| API | Description | Available in clients |
```

Comment:
I'm not sure that the table is the best way to render this content. On smaller screens (like my laptop, for example) the third column is cutoff from view entirely (you have to know to manually scroll to the right to see content in the third column, and when you do so, content in the first column scrolls out of view.

Suggest that you reformat this content such that it doesn't appear within a 3-column table. Perhaps consider making each 'category/area' of changes an H2, and under that list each API as an H3, then under each H3, include the API's **Description** and **Available in** info? for example:

## Attachments

### [AttachmentContent](/javascript/api/outlook/office.attachmentcontent)

Added a new object that represents the content of an attachment.

**Available in**: Outlook 2019 for Windows (Office 365 subscription)

### [Office.context.mailbox.item.addFileAttachmentFromBase64Async](office.context.mailbox.item.md#addfileattachmentfrombase64asyncbase64file-attachmentname-options-callback)

Added a new method that allows you to attach a file represented as a base64 encoded string to a message or appointment.

**Available in**: Outlook 2019 for Windows (Office 365 subscription)

...

The markdown for this example:

```
## Attachments

### [AttachmentContent](/javascript/api/outlook/office.attachmentcontent)

Added a new object that represents the content of an attachment.

**Available in**: Outlook 2019 for Windows (Office 365 subscription)

### [Office.context.mailbox.item.addFileAttachmentFromBase64Async](office.context.mailbox.item.md#addfileattachmentfrombase64asyncbase64file-attachmentname-options-callback)

Added a new method that allows you to attach a file represented as a base64 encoded string to a message or appointment.

**Available in**: Outlook 2019 for Windows (Office 365 subscription)
```

Reply:
@kbrandl: Thanks for the suggestion. The only change I made was to set them to H3 and H4 because these are already under an H2.